roles/st2chatops/vars/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+supported_hubot_adapter:
+ - slack
+ - hipchat
+ - xmpp
+ - flowdock
+ - yammer
+ - spark
+ - irc
```

Comment:
Also `mattermost` adapter.
Just to be one step ahead because of https://github.com/StackStorm/st2chatops/pull/65
I'm sure we'll forget to add it later.